modules/debug_cmd/debug_cmd.c

```diff
@@ -226,6 +363,7 @@ static const struct cmd debugcmdv[] = {
 {"modules",     0,       0, "Module debug",           mod_debug           },
 {"netstat",    'n',      0, "Network debug",          cmd_net_debug       },
 {"play",        0, CMD_PRM, "Play audio file",        cmd_play_file       },
+{"fileinfo",    0, CMD_PRM, "Audio file info",        cmd_fileinfo        },
```

Comment:
consider adding "au" prefix since it is audio related.

au - audio
vid - video